eventtracking.js

```diff
@@ -0,0 +1,44 @@
+const MIXPANEL_TOKEN = '9aa8926fbcb03eb5d6ce787b5e8fa6eb';
+var mixpanel = require('mixpanel').init(MIXPANEL_TOKEN);
+const homedir = require('os').homedir();
+const fs = require('fs');
+const path = require('path');
+const uuid = require('uuid');
+
+const track = () => {
+    const nearPath = path.join(homedir, '.near-config');
+    console.log(nearPath)
+    if (!fs.existsSync(nearPath)) { // TODO: make a util for .near
+        //create near path if it doesn't already exist
+        fs.mkdirSync(nearPath);
+    }
+
+    const shellSettingsPath = path.join(nearPath, 'shellSettings');
+    if (!fs.existsSync(shellSettingsPath)) {
+        //Ask if it's ok to collect feedback
+        console.log("We would like to collect data on near shell usage to improve developer experience. Would you like to opt in?")
```

Comment:
I think it would be great to use inquirer here:
https://www.npmjs.com/package/inquirer

I very much plan to use that at some point with `near-shell`

Reply:
Unless we actually need it's features (selecting from list, etc), it makes sense to just use built-in https://nodejs.org/api/readline.html

At least let's make sure only one of these gets used.

Reply:
I agree with only one way to get input. I am planning to use whatever is in login when I make this production quality (needs a couple of passes)